### PR TITLE
📚 Correct a few grammar pieces in the README

### PR DESCRIPTION
**Pull request by andygrunwald**
# Proposed Changes

This PR fixes a few smaller grammar glitches in the README.
Makes it easier to read :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ based on the following:
 
 ## Support
 
-Got questions?
+Have you got any questions?
```
Comment (frenck) [inline README.md]: Please revert this.

Comment (andygrunwald) [reply README.md]: Sure. Done.
Can you shortly explain the reasoning behind this?

Comment (frenck) [reply README.md]: I want to have it short, simple and consistent.

Comment (andygrunwald) [reply README.md]: Thanks. Change applied.

**Pull request merged**